docs/introduction/how-tos/quickstart.md

```diff
@@ -0,0 +1,66 @@
+Quickstart Guide
+================
+
+Local Install
+----------------
+
+### 1. Run minikube.sh
+
+```bash
+deployment/minikube/minikube.sh
+```
+
+### 2. Prepare Siembol Config Repository
+
+#### 1. Fork Siembol Config Repository
+
+1. Go to https://github.com/G-Research/siembol-config
+1. Fork into your own organization or personal account
+
+#### 2. Update application.properties
+
+Edit `deployment/helm/config-editor/resources/application.properties`
+
+Change <your_github_org_or_username> in these lines: 
+
+```
+config-editor.services.alert.config-store.git-user-name=<your_github_org_or_username>
+config-editor.services.alert.config-store.store-repository-name=<your_github_org_or_username>/siembol-config
+config-editor.services.alert.config-store.release-repository-name=<your_github_org_or_username>/siembol-config
+```
+
+### 3. Prepare git
+
+#### 1. Create Git Secret
+
+Generate a token
+1. Go to https://github.com/settings/tokens
+2. Click Generate Token
+4. Select "repo - Full control of private repositories" scope
+5. Hit "Generate token"
+6. Copy token value to a file named `git_token`
+
+#### 2. Generate a secret for your git API Token:
+
+This creates a Kubernetes secret for the Config Editor to interact with git.
+
+```bash
+kubectl create secret generic siembol-config-editor-rest-secrets --from-file=git_token -n siembol
```

Comment:
This ends up creating a key `git_token` in the secret whilst the helm chart expects a key `git`

Reply:
yep, reverted to just `git`